kpi/tests/api/v2/test_api_submissions.py

```diff
@@ -527,6 +530,67 @@ def test_duplicate_submission_by_anotheruser_shared_allowed(self):
         self._check_duplicate(response)
 
 
+class BulkUpdateSubmissionsApiTests(BaseSubmissionTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.submission_url = reverse(
+            self._get_endpoint('submission-bulk-update-submissions'),
+            kwargs={
+                'parent_lookup_asset': self.asset.uid,
+            },
+        )
+        self.updated_submission_data = {
+            'submission_ids': ['1', '2'],
+            'q1': '🕺',
```

Comment:
:smile:

Reply:
I have a good laugh every now and then looking through your tests, so I thought I should carry on the tradition 🤡